content/services/hosting/hosting_details.rst

```diff
@@ -17,124 +17,112 @@ Virtualization
     :align: right
```

Comment:
@ramereth Up top it says "Hosting Detail" not "Hosting Details" and should probably be changed while we're here.